File: sdks/csharp/src/NovaBilling/Tenants/Requests/TestSmtpTenantsRequest.cs
using System.Text.Json.Serialization;
using NovaBilling.Core;

namespace NovaBilling;

[Serializable]
public record TestSmtpTenantsRequest
{
    /// <summary>
    /// Recipient email address
    /// </summary>
    [JsonPropertyName("to")]
    public required string To { get; set; }

    /// <inheritdoc />
    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}

File: sdks/php/src/Tenants/Requests/TestSmtpTenantsRequest.php
<?php

namespace NovaBilling\Tenants\Requests;

use NovaBilling\Core\Json\JsonSerializableType;
use NovaBilling\Core\Json\JsonProperty;

class TestSmtpTenantsRequest extends JsonSerializableType
{
    /**
     * @var string $to Recipient email address
     */
    #[JsonProperty('to')]
    public string $to;

    /**
     * @param array{
     *   to: string,
     * } $values
     */
    public function __construct(
        array $values,
    ) {
        $this->to = $values['to'];
    }
}
